toolchain/lowering/lowering_handle.cpp

```diff
@@ -123,9 +122,16 @@ auto LoweringHandleCall(LoweringFunctionContext& context,
   for (auto ref_id : context.semantics_ir().GetNodeBlock(refs_id)) {
     args.push_back(context.GetLocalLoaded(ref_id));
   }
-  auto* value =
-      context.builder().CreateCall(function, args, function->getName());
-  context.SetLocal(node_id, value);
+  if (function->getReturnType()->isVoidTy()) {
+    context.builder().CreateCall(function, args);
+    // TODO call GetType with empty tuple type.
+    context.SetLocal(node_id, context.builder().CreateAlloca(
+                                  context.GetType(SemanticsTypeId::TypeType),
```

Comment:
If you want to leave fixing this as a TODO, I think that's okay, but let's use an anonymous struct here instead of calling GetType for TypeType. This should not rely on the implementation of TypeType being equivalent to an empty tuple type.

Reply:
Please restore the TODO. The current approach should be regarded as a workaround.